themes/default/content/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md

```diff
@@ -3,7 +3,7 @@ title: "Announcing Pulumi 0.15"
 date: "2018-08-15"
 meta_desc: "Pulumi can now deploy and manage Kubernetes resources using the same familiar programming model supported for AWS, Azure, and Google Cloud Platform."
 authors: ["luke-hoban"]
-tags: ["Pulumi", "New-Features", "CI/CD"]
+tags: ["New-Features", "continuous-delivery"]
```

Comment:
Are we lower-casing these by default now?

Reply:
The blog system lowercases them by default, so it doesn't really matter what's in code. My find-and-replace defaults to lowercase as well.